modules/vaos/app/serializers/vaos/clinic_serializer.rb

```diff
@@ -10,6 +10,14 @@ class ClinicSerializer
     attributes :site_code,
                :clinic_id,
                :clinic_name,
-               :clinic_friendly_location_name
+               :clinic_friendly_location_name,
+               :primary_stop_code,
+               :secondary_stop_code,
+               :direct_scheduling_flag,
+               :display_to_patient_flag,
+               :institution_name,
+               :institution_code,
+               :object_type,
```

Comment:
Object type and link I think could be left off, since they won't really mean anything to someone consuming the api through va.gov. Not sure if you're doing that with other endpoints, though.